docs/client/python/pyhive.md

```diff
@@ -0,0 +1,70 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# PyHive
+
+[PyHive](https://github.com/dropbox/PyHive) is a collection of Python DB-API and SQLAlchemy interfaces for Hive. PyHive is able to connect with Kyuubi server as Kyuubi server supports serving in thrift protocol as HiveServer2.
+
+PyHive is now marked as [no-longer maintained](https://github.com/dropbox/PyHive/commit/d199a1bd55c656b5c28d0d62f2d3f2e6c9a82a54) and staying on HiveServer2 Thrift bindings v11 which is compatible with Kyuubi server.
```

Comment:
this line is unnecessary

Reply:
OK, removed. My intention was to warn users who cares about end of support situation and maybe consider any other python lib approach.

Reply:
> My intention was to warn users who cares about end of support situation

the status of a third-party project is always unforeseeable, we can avoid making statements for them.

> any other python lib approach.

I hope its name will be pykyuubi:)

Reply:
Got your point. Avoid comments on other project. I have one sec hesitating to whether fill this PR as I found out PyHive is not maintained anymore. But after all , it's handy to use and approach to kyuubi source.

As for `pykyuubi`, I think we will have one in far future as we introduce more and more protocols and FE (like FlighSQL) libs for different usages. It must provide unique improvement other than just wrap some libs in it. Looking forward to it. 😄